patterns/footer.php

```diff
@@ -80,7 +80,26 @@
 
 <!-- wp:group {"align":"full","style":{"border":{"top":{"width":"1px"}}},"backgroundColor":"background","className":"course-negative-space-footer","layout":{"type":"constrained","contentSize":""}} -->
 <div class="wp-block-group alignfull course-negative-space-footer has-background-background-color has-background" style="border-top-width:1px"><!-- wp:paragraph {"align":"center","style":{"typography":{"letterSpacing":"0.02em"},"spacing":{"padding":{"top":"80px","bottom":"80px"}}},"backgroundColor":"background","fontSize":"xx-small","fontFamily":"system"} -->
-<p class="has-text-align-center has-background-background-color has-background has-system-font-family has-xx-small-font-size" style="padding-top:80px;padding-bottom:80px;letter-spacing:0.02em"><span style="text-decoration: underline;">Course Theme</span> by <a href="https://senseilms.com/">Sensei</a>, Powered by <span style="text-decoration: underline;">WordPress</span>.</p>
```

Comment:
@Imran92  is this supposed to be translated or this note we leave untranslated ? 

>Course Theme by [Sensei](https://senseilms.com/), Powered by [WordPress](https://wordpress.org/).

Reply:
Based on [this](https://github.com/Automattic/themes/blob/696e9387df923b9fec84dbf2ad513e887822b176/pixl/patterns/footer.php#L23-L25), I think we should probably translate this one too